tests/conftest.py

```diff
@@ -71,9 +71,8 @@ def mock_provider() -> Mock:
 
 @fixture(scope="session")
 def example_db_engine(example_db_provider: Callable[[], Database]) -> Engine:
-    with app.app_context():
-        with example_db_provider().get_sqla_engine() as engine:
-            return engine
+    with example_db_provider().get_sqla_engine() as engine:
```

Comment:
`example_db_provider` is completely mocked and thus doesn't use the app context.